src/connector/src/source/google_pubsub/split.rs

```diff
@@ -0,0 +1,28 @@
+use anyhow::anyhow;
+use bytes::Bytes;
+use serde::{Deserialize, Serialize};
+
+use crate::source::SplitMetaData;
+
+#[derive(Clone, Serialize, Deserialize, Debug, PartialEq, Hash)]
+pub struct PubsubSplit {
+    pub(crate) topic: String,
+    pub(crate) subscription: String,
+}
+
+impl PubsubSplit {}
+
+impl SplitMetaData for PubsubSplit {
+    fn id(&self) -> String {
+        // for now, until we figure out how we take care of IDs here
+        format!("{}", 0)
+    }
```

Comment:
there will be multiple readers run in parallel and every reader subscribe to one partition, which is manually set in your impl. For example, if you set `split_count` to 3, the id here can be one of 0, 1, 2

Reply:
Thanks, addressed in 402e285b762665a2b5246bd89128249b4273a8cc